# Parameters for real environment
FOV: 62 
camera_topic_realsense: "/camera/depth/image_rect_raw"

# Parameters for simulated environment
camera_topic_gazebo: "camera/depth/image_raw"
FOV_gazebo: 55

# Parameters common for simulated and real environment
crop_size: 300
camera_topic_info: "/camera/depth/camera_info"
camera_topic_ssd512: "/obj_img"

# Check FOV in https://www.intel.com/content/dam/support/us/en/documents/emerging-technologies/intel-realsense-technology/Intel-RealSense-D400-Series-Datasheet.pdf
# pag. 51 in Depth FOV VGA (degrees) - Consider vertical VGA FOV
# Min depth for D435 = 175mm